bin/carl.ml

```diff
@@ -544,10 +544,23 @@ module CLI = struct
     Arg.(
       value & opt (some request_conv) None & info [ "X"; "request" ] ~doc ~docv)
 
-  let cacert =
+  let cacertfile =
+    let cert_conv =
+      let parse s = Ok (Cert.Filepath s) in
+      Arg.conv ~docv:"method" (parse, Cert.pp)
+    in
+    let doc = "CA certificate to verify peer against" in
+    let docv = "file" in
+    Arg.(value & opt (some cert_conv) None & info [ "cacertfile" ] ~doc ~docv)
```

Comment:
I thought I'd written something down in a comment, but I can't find it now, so I might have deleted before submitting.

My motivation for `carl` was to follow `curl`'s command line API as close as possible. Unfortunately they only have a `--cacert` option, and these would not be in line with that API. Do you think we could revert this, and only support the certfile? I'ts a little cumbersome to write an entire PEM string in the command line, anyway.